rst_files/multiplicative_functionals.rst

```diff
@@ -154,9 +154,9 @@ Setup
 
     function multiplicative_decomp(A, B, D, F, ν)
         H, g = additive_decomp(A, B, D, F)
-        ν_tilde = ν + 0.5 * H^2
+        ṽ = ν + 0.5 * H^2
```

Comment:
Hey @PooyaFa, I think these are actually `nu`s and not `v`s? In that case, we should revert this, since the glyphs only work for Latin letters and not Greek.

Reply:
To be clear, if something is `nu` in the math, I think we can leave it as `nu` instead of coercing it to be `v`.

Reply:
Sorry, I couldn't tell the difference between v and nu.

Reply:
Don't worry about it